webviz_ert/plugins/_observation_analyzer.py

```diff
@@ -12,14 +12,21 @@
 
 
 class ObservationAnalyzer(WebvizErtPluginABC):
-    def __init__(self, app: dash.Dash, project_identifier: str):
+    def __init__(self, app: dash.Dash, project_identifier: str, beta: bool = False):
         super().__init__(app, project_identifier)
         self.set_callbacks(app)
+        self.beta = beta
 
     @property
     def layout(self) -> Component:
         return html.Div(
             [
+                html.Div(
```

Comment:
I wonder if we should do something more to this text - feels a bit strange only heaving BETA as header... 🤔  Would it make sense having a descriptive text instead / in addition? 
```
* This page is considered a beta version and could be changed or removed. You are encouraged to use it and give feedback to us regarding functionality and / or bugs. 
```